src/map/Mapbox.ts

```diff
@@ -83,6 +88,119 @@ export default class Mapbox {
         this.drawSelectedPath(selectedPath)
     }
 
+    showPathDetails(selectedPath: Path) {
+        if (selectedPath.points.coordinates.length === 0)
+            return;
+        if (!this.map.hasControl(this.heightgraph))
+            this.map.addControl(this.heightgraph, 'bottom-right');
+        const elevation = this.createFeatureCollection(
+            'Elevation [m]',
+            [this.createFeature(selectedPath.points.coordinates, 'elevation')]
+        );
+        const pathDetails = Object.entries(selectedPath.details).map(([detailName, details]) => {
+            const points = selectedPath.points.coordinates;
+            const features = [];
+            for (let i = 0; i < details.length; i++) {
+                let [from, to, value] = details[i];
+                if (typeof value === 'undefined' || value === null)
+                    value = 'Undefined'
+                features.push(this.createFeature(points.slice(from, to + 1), value));
+            }
```

Comment:
I'd shorten this like the following:
```
const features = details.map(([from, to, value = 'Undefined']: [number, number, any]) => 
                this.createFeature(points.slice(from, to + 1), value))
```
If value is always either string or not set, the last type of the triple could be string